ncc/examples/wu.c

```diff
@@ -0,0 +1,256 @@
+#include <assert.h>
+#include <stdlib.h>
+#include <uvm/graphics.h>
+#include <uvm/syscalls.h>
+#include <uvm/utils.h>
+
+
+size_t FRAME_WIDTH = 400;
+size_t FRAME_HEIGHT = 400;
+
+u32 frame_buffer[160000];
+
+// Current mouse pointer position
+size_t pos_x = 200;
+size_t pos_y = 200;
+
+void
+carefree_alpha_blend_plot_pixel(u32* dest, size_t dest_stride, u64 x, u64 y, u32 color, u8 alpha)
```

Comment:
Here I don't understand why `carefree` :sweat_smile:

Do some versions of alpha blending worry more than others :thinking:

Reply:
It's just to remind myself that this function doesn't check it's arguments, it will just blithely write to any old address if you give it bad args.  The image in my mind is the scene from *Austin Powers* where Dr. Evil is like, "We'll just close the doors and assume everything went to plan." :)

Reply:
Should probably use the term `unsafe_`